### DISCUSSION TITLE
Datetime issue with Stadtreinigung Hamburg

### DISCUSSION THREAD

**Discussion by clonefx** [mampfes/hacs_waste_collection_schedule#2662] [Q&A] [answered]
Hello!

since two days I'm facing an issue with the integration of Stadtreinigung Hamburg. All worked fine before, then the dates suddenly disappeared. When trying to re-configure the integration, I get the following error message:

"can't compare datetime.datetime to datetime.date"

Any ideas? Pulling the dates from their website directly works fine.

Thanks

**Comment by 5ila5** [comment, marked as answer]
This is a problem with one version of the icalevents librabry. This is already fixed in WCS version 2.2.0 I released yesterday. Just update Waste Collection Schedule

**Reply by clonefx** [reply]
Thank you very much for your prompt reply, that solved the issue.
